examples/common/config.yaml

```diff
@@ -3,38 +3,53 @@
 # https://www.envoyproxy.io/docs/envoy/latest/configuration/configuration
 static_resources:
   listeners:
-    - address:
-        socket_address: {address: 0.0.0.0, port_value: 9001, protocol: TCP}
-      filter_chains:
-        - filters:
-            - name: envoy.http_connection_manager
-              config:
-                stat_prefix: client
-                route_config:
-                  virtual_hosts:
-                    - name: all
-                      domains: ["*"]
-                      routes:
-                        - match: {prefix: "/"}
-                          route: {cluster: hello_world_api}
-                http_filters:
-                  - name: envoy.router
+  - name: listener_0
```

Comment:
nit: no real need to name the listener here

Reply:
Removed